src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -1725,10 +1725,16 @@ public Response getLinks(@Context ContainerRequestContext crc, @PathParam("id")
             List<Dataverse> dvsThatLinkToThisDatasetId = dataverseSvc.findDataversesThatLinkToThisDatasetId(datasetId);
             JsonArrayBuilder dataversesThatLinkToThisDatasetIdBuilder = Json.createArrayBuilder();
             for (Dataverse dataverse : dvsThatLinkToThisDatasetId) {
-                dataversesThatLinkToThisDatasetIdBuilder.add(dataverse.getAlias() + " (id " + dataverse.getId() + ")");
+                JsonObjectBuilder entry = Json.createObjectBuilder();
+                entry.add("id", dataverse.getId());
+                entry.add("alias", dataverse.getAlias());
+                entry.add("displayName", dataverse.getDisplayName());
+                dataversesThatLinkToThisDatasetIdBuilder.add(entry);
             }
             JsonObjectBuilder response = Json.createObjectBuilder();
-            response.add("dataverses that link to dataset id " + datasetId, dataversesThatLinkToThisDatasetIdBuilder);
```

Comment:
Should we keep this line in for backward compatibility?

Reply:
OK, I'll put it back.

Reply:
@pdurbin Sorry, do you mean the key of the object ("dataverses that link to dataset id X") or the value (e.g. "crc990 (id 18802)")?

Reply:
Well, I think you need both for backward compatibility. We'd want `dataversesThatLinkToThisDatasetIdBuilder` to have whatever value it has always had.

Reply:
It was decided today that there is no need for backward compatibility since the api was never documented